src/sass/components/apl.scss

```diff
@@ -508,6 +523,30 @@
     }
   }
 
+  .apl-list-item {
+    &.item-button-right {
+      .button-right-container {
+        position: absolute;
+        top: 16px;
+        right: 16px;
+        bottom: 16px;
+
+        display:flex;
+        flex-direction: column;
+        justify-content: center;
+
+        .button-right {
+          font-weight: bold;
+          background-color: #EFEFF4;
+          padding: 4px 16px;
+          border-radius: 23px;
+          width: $button-right-width + px;
```

Comment:
Is this proper syntax?

Reply:
@holemcross, can you explain which proper syntax do you expect?